src/json_parser.zig

```diff
@@ -360,12 +360,6 @@ pub const PackageJSONVersionChecker = struct {
     allocator: std.mem.Allocator,
     depth: usize = 0,
 
-    found_version_buf: [1024]u8 = undefined,
-    found_name_buf: [1024]u8 = undefined,
-
-    found_name: []const u8 = "",
-    found_version: []const u8 = "",
-
```

Comment:
We put these in writers now, which is an incremental change for the better in my view. We still should check/fix more places where overflow can occur though.